pkg/config/templates/rancherd-21-harvester-clusternetworks.yaml

```diff
@@ -0,0 +1,20 @@
+bootstrapResources: {{ if not .ClusterNetworks -}} [] {{- else }}
+{{- range $name, $network := .ClusterNetworks }}
+- apiVersion: harvesterhci.io/v1beta1
+  kind: ClusterNetwork
+  enable: true
```

Comment:
Why is it always true?

Reply:
let me add an option for "enable"

i thought if they will put config for installer, and they will only want to enable it right away

Reply:
We can set it as `true` if the VLAN ID and default physical nic are not empty. What do you think?

Reply:
i'm thinking not to detect "config" field for all kind of clusternetworks.
because that will need to setup lots of validate function for all kinds in installer if we have further network type to support. it will cause we have two place that need to be fixed if we have some change (installer and network-controller)

Maybe we can setup "false" as default and put some words in docs for notice?

Reply:
Agree